packages/lage/src/command/graph.ts

```diff
@@ -1,17 +1,21 @@
+import { Config } from "../types/Config";
 import { getWorkspaceRoot, getPackageInfos, createPackageGraph } from "workspace-tools";
+import { getConfig } from "../config/getConfig";
 
-export async function graph(cwd: string) {
+export async function graph(cwd: string, config: Config) {
+  const fs = require("fs");
   const root = getWorkspaceRoot(cwd);
   if (!root) {
     throw new Error("This must be called inside a codebase that is part of a JavaScript workspace.");
   }
   const allPackages = getPackageInfos(root);
+  const pattern = getConfig(cwd).scope;
```

Comment:
If scope isn't set, what happens? Should we supply the other flags if the pattern is empty?

Reply:
I tested this within createPackageGraph and it was returning the entire graph, however, the command returns an empty graph -- plus it looks like there are errors in getting the development and peer dependencies using the command flags. I will be looking into this in more detail!

Reply:
I think getConfig might fill out scope to something by default?

Reply:
I just added an if else statement that checks if the pattern is empty/undefined and sets 
* the namePatterns as all packages
* includeDependencies, includeDependents, withDevDependencies, and withPeerDependencies to true 
so if there is no scope provided, it shows the entire graph!